src/withStyles.jsx

```diff
@@ -28,6 +31,10 @@ function baseClass(pureComponent) {
   return React.Component;
 }
 
+const contextTypes = {
+  [CHANNEL]: brcastShape,
```

Comment:
it's unfortunate we're not using `withDirection` here :-/

Reply:
I mean it would add an extra virtual DOM node to ... every single component in our tree... which doesn't seem good. :/

Reply:
Maybe we could export more things from react-with-direction to avoid as much duplication as possible?

Reply:
Perhaps we could export `directionProviderContextTypes`? the grabbing of the direction from the context is also a bit gross.

Reply:
I'll think about implementing this in a follow up

Reply:
Yes, we should definitely do that at a minimum.